src/org/ensembl/healthcheck/testcase/compara/MemberProductionValues.java

```diff
@@ -0,0 +1,71 @@
+package org.ensembl.healthcheck.testcase.compara;
+
+import org.ensembl.healthcheck.DatabaseRegistryEntry;
+import org.ensembl.healthcheck.DatabaseType;
+import org.ensembl.healthcheck.ReportManager;
+import org.ensembl.healthcheck.Team;
+import org.ensembl.healthcheck.testcase.AbstractTemplatedTestCase;
+import org.ensembl.healthcheck.util.SqlTemplate;
+import org.ensembl.healthcheck.util.DBUtils;
+
+public class MemberProductionValues extends AbstractTemplatedTestCase {
+
+	protected int totalCount;
+
+	public MemberProductionValues() {
+		setTeamResponsible(Team.COMPARA);
+		appliesToType(DatabaseType.COMPARA);
+		setDescription("Checks whether the gene_member_hom_stats table has been properly populated");
+	}
+
+	@Override
+	protected boolean runTest(DatabaseRegistryEntry dbre) {
+		SqlTemplate srv = getSqlTemplate(dbre);
+		boolean result = true;
+
+        getTotalCould(dbre);
+		result = computeMissingValues (dbre, totalCount);
+
+		return result;
+	}
+
+    //Get count
+	protected void getTotalCould(DatabaseRegistryEntry dbre) {
+		String sql = "SELECT gene_member_id FROM gene_member_hom_stats WHERE collection = 'murinae'";
+		totalCount = DBUtils.getRowCount(dbre.getConnection(), sql);
```

Comment:
Instead of changing a class attribute, `getTotalCould` could return the value (`totalCount`)